boards/BMS/Test/Src/Test_StateMachine.cpp

```diff
@@ -853,6 +860,58 @@ TEST_F(
         App_CanTx_GetPeriodicSignal_STATE(can_tx_interface));
 }
 
+// BMS-41
+TEST_F(BmsStateMachineTest, check_for_successful_minimum_duration_precharge)
+{
+    // Check that state transitions do not occur when the TS voltage is below
+    // 93% of the accumulator's maximum voltage while pre-charging capacitors to
+    // 90% of the accumulator's max voltage
```

Comment:
We should include an update to the specs describing the 93% behaviour before merging this PR